packages/conveyer/src/Conveyer.ts

```diff
@@ -425,12 +426,19 @@ class Conveyer extends Component<ConveyerEvents> {
     });
 
     this._axes = axes;
-    if (this._options.useDrag) {
-      axes.connect(this._options.horizontal ? ["scroll", ""] : ["", "scroll"], new PanInput(scrollAreaElement, {
+    if (options.useDrag) {
+      axes.connect(options.horizontal ? ["scroll", ""] : ["", "scroll"], new PanInput(scrollAreaElement, {
         inputType: ["mouse"],
         touchAction: "auto",
       }));
     }
+    if (options.useWheel && options.horizontal) {
+      axes.connect(["scroll", ""], new WheelInput(scrollAreaElement, {
```

Comment:
We don't need the feature to use the wheel in the scroll direction.
We need a feature that wheel in the side direction of the scroll direction.